src/main/java/org/example/ngevaticketmanagerspring/user/UserRepository.java

```diff
@@ -0,0 +1,9 @@
+package org.example.ngevaticketmanagerspring.user;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface UserRepository extends JpaRepository<User, Integer> {
```

Comment:
```suggestion
public interface UserRepository extends JpaRepository<User, Long> {
```
Wie oben erwähnt, hat Pascal eben Long festgelegt.